README.md

```diff
@@ -100,13 +100,13 @@ eredis_cluster:qk(["FLUSHDB"], "TEST").
 
 The directory contains a Makefile that uses rebar3.
 
-Setup a Redis cluster and start the tests using following commands:
+Setup Redis clusters and start the tests using following commands:
 
 ```bash
-make        # ... or rebar3 compile
-make start  # Start a local Redis cluster using Docker
-make test   # Run tests towards the cluster
-make stop   # Teardown the Redis cluster
+make compile # ... or rebar3 compile
```

Comment:
Only `make` (default target `all`) runs compile, dialyzer, xref and elvis. It could be good to mention this somewhere too (or possibly rename it to something like `make check` and let the default `make` target be `make compile`).

Reply:
Yes, `make check` might be good.
I had problems installing elvis on Mac, did not investigate deeply but a dependency "egithub 0.7.0 could no be found in any repo". Maybe elvis should not be required when just wanting to build and test.
@alexandremcosta, did you have the same problem?

Reply:
Well, it isn't required if you run e.g. `rebar3 compile`, but in other projects, `make` is only compilation. So it would be more consistent with other projects. Btw, I think `make check` should include running the tests too.